experiment/resources/runner-startup-script-template.sh

```diff
@@ -52,4 +53,5 @@ docker run \
 -e LOCAL_EXPERIMENT={{local_experiment}} \
 {% if not local_experiment %}--name=runner-container {% endif %}\
 --cap-add SYS_NICE --cap-add SYS_PTRACE \
+--security-opt seccomp=unconfined \
```

Comment:
So what would be the downsides of this?

Reply:
https://stackoverflow.com/questions/60840320/docker-50-performance-hit-on-cpu-intensive-code/61053027#61053027

No downsides imo, it's not a service exposed running on docker that has security boundaries